### PR TITLE
[SMALLFIX] fix checksytle

### PR DESCRIPTION
**Pull request by siyangy**

### CODE REVIEW

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java [hunk 1]
```diff
@@ -39,8 +39,10 @@
 public final class AlluxioFuse {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private static Configuration sConfiguration;
-  private AlluxioFuse(){}
-  
+
+  private AlluxioFuse() {
```
Comment (yupeng9) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java]: Can you also add a comment of 'prevent instantiation'?

**Comment by yupeng9** [comment]
Thanks for the improvement. I left one minor comment

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java (hunk 1 above)
Comment (siyangy) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java]: done

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**